addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -330,8 +330,8 @@ def _process_step_forward_operator(self, discuss_channel):
 
         if discuss_channel.livechat_channel_id:
             human_operator = discuss_channel.livechat_channel_id._get_operator(
-                lang=discuss_channel.livechat_visitor_id.lang_id.code,
-                country_id=discuss_channel.country_id.id
+                lang=discuss_channel.livechat_visitor_id.lang_id.code if hasattr(discuss_channel, "livechat_visitor_id") else None,
+                country_id=discuss_channel.country_id.id if hasattr(discuss_channel, "country_id") else None
```

Comment:
I don't think we need to guard on `country_id`, as this field is defined in `im_livechat`: https://github.com/odoo/odoo/blob/17.0/addons/im_livechat/models/discuss_channel.py#L28

Reply:
Removed the guard on `country_id`.